### PR TITLE
c++ docs page

### PR DESCRIPTION
**Pull request by kenny-statsig**
While writing docs, noticed a couple things the C++ SDK is lacking
    - Pass value and metadata to `logEvent`
    - DynamicConfic `get` with default accessor
    - Environment

### CODE REVIEW

File: docs/getting-started.mdx [hunk 1]
```diff
@@ -264,6 +264,11 @@ export const SDKCard = ({ language, image, link }) => (
         image="https://latest.www.statsig.com/images/sdks/sdk_python.png"
         link="/server/pythonSDK"
       ></SDKCard>
+      <SDKCard
+        language="C++"
+        image="/img/sdk/sdk_cpp.png"
+        link="/server/cppSDK"
+      ></SDKCard>
```
Comment (tore-statsig) [inline docs/getting-started.mdx:271]: wow

File: docs/server/cpp/_reference.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+          a.custom == b.custom &&
+          a.privateAttribute == b.privateAttribute &&
+          a.statsigEnvironment == b.statsigEnvironment &&
+          a.customIDs == b.customIDs;
+};
+```
+
+### Options
+
+```cpp
```
Comment (tore-statsig) [inline docs/server/cpp/_reference.mdx:36]: do we need to add cpp to the downloaded language highlighting maps?

**Comment by jasonwzm** [comment]
🚀 Deployed on https://64c441986d214d0f54be2112--cozy-fox-0defec.netlify.app

**Review by tore-statsig — APPROVED** [review]
you need some sort of warning or notes about not supporting ID lists/layers/etc

now that I said that, I realize that erlang doesnt have that warning :p

**Pull request merged**